app/views/users/show.html.erb

```diff
@@ -1,2 +1,17 @@
-<h1>Users#show</h1>
-<p>Details of user</p>
+<%= render partial: 'shared/header' %>
+<%= render partial: 'user', locals: {user: @user} %>
+<div class="border border-dark">
+<h2 class="fs-4">Bio</h2>
+<p>
+  <%= @user.bio %>
+</p>
+</div>
+
+
+<div>
+  <% @user_posts.each do |post| %>
```

Comment:
- [ ] This one is not OPTIONAL. I would like to remind us of the method in the user model that returns the three most recent posts. That is what we are expected to use in this view according to the wireframe. Please use that method to only display the three most recent posts and not all user's posts.

Reply:
💯 This issue is resolved successfully ✔️